config/default/manager_auth_proxy_patch.yaml

```diff
@@ -3,8 +3,8 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: manager
-  namespace: default
+  name: controller-manager
+  namespace: system
```

Comment:
The name and namespace had been altered in https://github.com/improbable-eng/etcd-cluster-operator/pull/27/files#diff-4693cc41eb3660fc1d3af8a11304f44c 
(I can't remember why)

But that breaks all the other kustomize patches that come with kubebuilder, 
so I've reverted that change.

Also updated the e2e tests accordingly.